cmd/policy-assistant/pkg/kube/labelselector.go

```diff
@@ -107,14 +107,12 @@ func LabelSelectorTableLines(selector metav1.LabelSelector) string {
 	}
 	var lines []string
 	if len(selector.MatchLabels) > 0 {
-		lines = append(lines, "Match labels:")
```

Comment:
Can we keep this and `Match expressions: `? Similarly to my previous comment, leaving this will make the output clearer imo

Reply:
Or feel free to change the wording, and we can assess outputs for both the MatchLabels case and MatchExpressions